extensions/python/tests/PythonManifestTests.cpp

```diff
@@ -101,7 +101,7 @@ TEST_CASE("Python processor's description is part of the manifest") {
     REQUIRE(getNode(MyPyProc->children, "type").value == "org.apache.nifi.minifi.processors.MyPyProc");
 
     auto& rels = getNode(MyPyProc->children, "supportedRelationships").children;
-    REQUIRE(rels.size() == 2);
+    REQUIRE(rels.size() == 3);
```

Comment:
What changed here?

Reply:
I added the "original" default relationship to the python processors as the NiFi python processors have it.